runtime/src/plugin.rs

```diff
@@ -904,6 +904,10 @@ impl Plugin {
                         )));
                     }
                 }
+            } else if let Err(e) = &res {
+                if e.is::<wasmtime::Trap>() {
+                    rc = 134; // EXIT_SIGNALED_SIGABRT
+                }
```

Comment:
I wonder if we should do something like:

```suggestion
                } else if rc == 0 {
                  rc = -1;
                }
```

in case it's possible for there to be another case where an error is returned but rc == 0. But maybe that's not needed.

Reply:
In bf95d06 I reworked it to default to -1 and avoid hiding errors that can happen first. Extism errors can happen before wasmtime errors, which happen before retrieving output errors.